lib/core-integration/src/Test/Integration/Scenario/API/Shelley/TransactionsNew.hs

```diff
@@ -78,19 +106,105 @@ spec :: forall n.
     , PaymentAddress n IcarusKey
     ) => SpecWith Context
 spec = describe "NEW_SHELLEY_TRANSACTIONS" $ do
-    it "TRANS_NEW_CREATE_01x - No payload" $ \ctx -> runResourceT $ do
-        let initialAmt = 3*minUTxOValue
-        wa <- fixtureWalletWith @n ctx [initialAmt]
-
+    it "TRANS_NEW_CREATE_01a - Empty payload is not allowed" $ \ctx -> runResourceT $ do
+        wa <- fixtureWallet ctx
         let emptyPayload = Json [json|{}|]
 
         rTx <- request @(ApiConstructTransaction n) ctx
             (Link.createUnsignedTransaction @'Shelley wa) Default emptyPayload
         verify rTx
             [ expectResponseCode HTTP.status403
+            , expectErrorMessage errMsg403InvalidConstructTx
+            ]
+
+    it "TRANS_NEW_CREATE_01b - Validity interval only is not allowed" $ \ctx -> runResourceT $ do
+        wa <- fixtureWallet ctx
+        let validityInterval = Json [json|{
+                "validity_interval": {
```

Comment:
@paweljakubas We should also add a simple validation (invalid before <= invalid hereafter) of the validity interval to the API JSON parser.